app/decorators/user_decorator.rb

```diff
@@ -49,21 +49,6 @@ def lockout_time_remaining
     (lockout_period - (Time.zone.now - user.second_factor_locked_at)).to_i
   end
 
-  def confirmation_period_expired_error
-    I18n.t('errors.messages.confirmation_period_expired', period: confirmation_period)
-  end
-
-  def confirmation_period
```

Comment:
This was used, but only in tests, the live usage is actually in `ConfirmationEmailPresenter`